src/components/Table/TableViewDropdown/__snapshots__/TableViewDropdown.story.storyshot

```diff
@@ -47,7 +47,7 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/Table
           >
             <span
               className="bx--list-box__label"
-              htmlFor="downshift-10-input"
```

Comment:
Any idea why this changed?

Reply:
Not sure why these changed.

Reply:
This is generated by downshift - the id counter is added to on every new instance of downshift (when snaps are generated) if an id isn't passed in. We are passing the ID into the carbon component, so I'm not sure why it's using the generated one instead of the one we pass in.

https://github.com/downshift-js/downshift/blob/a71005bd879d05d7dcb892d1e0dc5c6ca74e9d39/src/utils.js#L113-L119

I don't think there's any way for us to prevent it other than resetting the counter before snaps are ran. Either way, can ignore for now